### DISCUSSION TITLE
Disabling "Tablet" disables tablet input outright in-game on Wayland

### DISCUSSION THREAD

**Discussion by 25pwn** [ppy/osu#16764] [Ideas] [answered]
According to https://github.com/ppy/osu/discussions/15580, disabling "Tablet" should simply disable the in-game tablet driver. On X11, this works as expected, but on Wayland, it disables tablet input outright in-game. Toggling "High precision mouse" has no effect.
I'm running a KDE Wayland session on Arch Linux.

**Comment by peppy** [comment, marked as answer]
please diagnose this at your end. we can't fix per installation linux issues. if X11 works for you and you can't find a solution, please use X11.

**Reply by 25pwn** [reply]
I found a solution, setting SDL_VIDEODRIVER=wayland fixes this.
